web/thesauruses/python/3/queues_stacks.json

```diff
@@ -0,0 +1,388 @@
+        "name": "Queue data type name",
+        "code": [
+          "list"
+        ]
+      },
+      "simple_queue_create_statement": {
+        "name": "Create a queue",
+        "code": [
+          "queue = ['One', 'Two', 'Three']"
+        ]
+      },
+      "simple_queue_create_copy_statement": {
+        "name": "Create a queue from existing queue",
+        "code": [
+          "new_queue = queue"
+        ]
+      },
+      "simple_queue_create_copy_from_list_statement": {
+        "name": "Create a queue from a list of items",
+        "code": [
+          "queue = mylist"
+        ]
+      },
+      "simple_queue_destroy_statement": {
+        "name": "Destroy/delete a queue",
+        "code": [
+          "queue.clear()"
+        ]
+      },
+      "simple_queue_data_structure": {
+        "name": "Data structure that backs the simple queue",
+        "code": [
+          "list"
+        ]
+      },
+      "circular_queue_import_statement": {
+        "name": "Import statement to add circular queue",
+        "not-implemented": "true"
+
+      },
+      "circular_queue_data_type": {
+        "name": "Queue data type name",
+        "not-implemented": "true"
+
+      },
+      "circular_queue_create_statement": {
+        "name": "Create a queue",
+        "not-implemented": "true"
+
+      },
+      "circular_queue_create_copy_statement": {
+        "name": "Create a queue from existing queue",
+        "not-implemented": "true"
+
+      },
+      "circular_queue_create_copy_from_list_statement": {
+        "name": "Create a queue from a list of items",
+                "not-implemented": "true"
+
+      },
+      "circular_queue_destroy_statement": {
+        "name": "Destroy/delete a queue",
+                "not-implemented": "true"
+
+      },
+      "circular_queue_data_structure": {
+        "name": "Data structure that backs the circular queue",
+                "not-implemented": "true"
+
+      },
+      "priority_queue_import_statement": {
+        "name": "Import statement to add priority queue",
+        "code": [
+          "from queue import PriorityQueue"
+        ]
+      },
+      "priority_queue_data_type": {
+        "name": "Queue data type name",
+        "code": [
+          "queue.PriorityQueue"
+        ]
+      },
+      "priority_queue_create_statement": {
+        "name": "Create a queue",
+        "code": [
+          "queue = PriorityQueue()"
+        ]
+      },
+      "priority_queue_create_copy_statement": {
+        "name": "Create a queue from existing queue",
+        "code": [
+          "new_queue = queue"
+        ]
+      },
+      "priority_queue_create_copy_from_list_statement": {
+        "name": "Create a queue from a list of items",
+        "code": [
+          "from queue import PriorityQueue\nimport itertools\n\nqueue = PriorityQueue()\n\npriorities = [1, 2, 3]\n\nmylist = ['One', 'Two', 'Three']\n\nfor priority, item in list(zip(priorities, mylist)):\n\tqueue.put((priority, item))"
```

Comment:
This is sort of inventing the functionality for this, which isn't what you should do. This should be "not-implemented" as well.